content/en/docs/instrumentation/erlang/getting-started.md

```diff
@@ -267,3 +264,118 @@ iex(2)>
 {{< /tab >}}
 
 {{< /tabs >}}
+
+## Exporting to the OpenTelemetry Collector
+
+The [Collector](/docs/collector/) provides a vendor
+agnostic way to receive, process and export telemetry data. The package
+[opentelemetry_exporter](https://hex.pm/packages/opentelemetry_exporter)
+provides support for both exporting over both HTTP (the default) and GRPC to the
+collector which can then export Spans to a self-hosted service like Zipkin or
+Jaeger, as well as commercial services. See the [OpenTelemetry Collector's
+contrib
+repo](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter)
+for a full list of available exporters in the Collector.
```

Comment:
Two things here -- consider the following:

- Redirect the reader to the registry at https://opentelemetry.io/registry/?component=exporter instead.
- Rephrase as:
  > For a full list of available exporters, see <link>.

Reply:
Good point about the registry. I've updated the phrasing and the link.